content/applications/inventory_and_mrp/barcode/setup/hardware.rst

```diff
@@ -4,64 +4,78 @@ Set up your barcode scanner
+
+- **Bluetooth scanners** pair with a smartphone or tablet, making them an ideal cost-effective and
+  portable barcode scanner option. In this scenario, Odoo is installed on the smartphone, allowing
+  warehouse operators to handle operations, and check stock directly through their mobile devices.
+
+- **Mobile computer scanners** are mobile devices with a built-in barcode scanner. First, ensure the
+  device can run the Odoo mobile app properly. Recent models that use Android OS with the Google
+  Chrome browser, or Windows OS with Microsoft Edge, should work. However, testing is crucial due to
+  the variety of available models and configurations.
+
+Configuration
+=============
+
+When setting up the barcode scanner, ensure the following configurations are correctly set to ensure
+the scanner can interpret barcodes properly in Odoo.
 
 Keyboard layout
 ---------------
 
-.. image:: hardware/hardware01.png
-    :align: center
+When using a USB barcode scanner, match its keyboard layout with the operating system's layout for
+proper interpretation of characters. Generally, the scanning mode should be set to accept a USB
+keyboard (HID), with the language set based on the keyboard that is in use.
 
-An USB barcode scanner needs to be configured to use the same keyboard
-layout as your operating system. Otherwise, your scanner won't translate
-characters correctly (replacing a 'A' with a 'Q' for example). Most
-scanners are configured by scanning the appropriate barcode in the user
-manual.
+When configuring the keyboard layout for a Zebra scanner, scan the keyboard wedge barcode in the
+user manual for the barcode scanner.
+
+.. image:: hardware/keyboard-barcode.png
+   :align: center
+   :alt: Example of a user manual for keyboard layout.
 
 Automatic carriage return
 -------------------------
 
-By default, Odoo has a 50 milliseconds delay between each successive
-scan (it helps avoid accidental double scanning). If you want to
-suppress this delay, you can configure your scanner to insert a carriage
-return at the end of each barcode. This is usually the default
-configuration and can be explicitly configured by scanning a specific
-barcode in the user manual ('CR suffix ON', 'Apply Enter for suffix',
-etc.).
+Odoo has a default 50-millisecond delay between scans to prevent accidental double scanning. To
+synchronize with the barcode scanner, set it to include a *carriage return* (:dfn:`character like
+the "Enter" key on a keyboard`) after each scan. Odoo interprets the carriage return as the end of
+the barcode input; so Odoo accepts the scan, and waits for the next one.
+
+Typically, on the scanner, a carriage return is included by default. Ensure it is set by scanning a
+specific barcode in the user manual, like 'CR suffix ON' or 'Apply Enter for suffix.'
+
+Zebra scanner
+-------------
+
+When using Zebra scanners, ensure the correct keystroke configurations are set to prevent errors.
```

Comment:
Would indicate that the correct settings are to follow in the documentation otherwise we're just saying "make sure your settings are correct" in a general (and unhelpful) sense  😄 

```suggestion
When using Zebra scanners, ensure the following keystroke configurations are set to prevent errors.
```